rfc/8/index.md

```diff
@@ -0,0 +1,1117 @@
+
+#### 10. Gallery / grid views
+
+It is useful to visualise similar images in a grid view where all images are visible as "thumbnails", which in the case of OME-Zarr can simply be the lowest resolution version of the data. Like this, users can have an overview of all the data and can then decide to "zoom in" on some datasets to explore them in higher resolution.
+
+Implementations of this concept include:
+- [Zarrcade](https://github.com/JaneliaSciComp/zarrcade)
+- [BioFile Finder](https://bff.allencell.org/)
+- [MoBIE grid views](https://mobie.github.io/tutorials/image_grids_and_tables.html)
+- [OME2024 NGFF challenge](https://ome.github.io/ome2024-ngff-challenge/)
+
+For example, [this table](https://docs.google.com/spreadsheets/d/1t5xB0p0zd2-a6ynV-JAuLJqs-mg-pFFikhfmQGZwRpI/edit?usp=sharing) defines a MoBIE grid view of three OpenOrganelle vEM images along with label images of mitochondria segmentation. It can be opened in MoBIE via the "Open Simple Collection Table" menu entry: 
+
+![MoBIE grid view](./assets/mobie_grid_view.jpg)
+
+
+
+<!--
+The next section is the "Background" section. This section should be at least
+two paragraphs and can take up to a whole page in some cases. The \*\*guiding goal
+of the background section\*\* is: as a newcomer to this project (new employee, team
+transfer), can I read the background section and follow any links to get the
+full context of why this change is necessary? 
+
+If you can't show a random engineer the background section and have them
+acquire nearly full context on the necessity for the RFC, then the background
+section is not full enough. To help achieve this, link to prior RFCs,
+discussions, and more here as necessary to provide context so you don't have to
+simply repeat yourself.
+
+-->
+
+## Proposal
+
+<!--
+The next required section is "Proposal". Given the background above, this
+section proposes a solution. This should be an overview of the "how" for the
+solution, but for details further sections will be used.
+-->
+
+This proposal adds collections to the OME-Zarr specification.
+"Collections" are groupings of "nodes".
+Nodes either reference OME-Zarr images (multiscales) or collections.
+Nested collections can also be inlined in the metadata.
+Nodes reference images or collections that are stored locally relative to the collection or remotely (using URLs).
+Arbitrary user or implementation metadata may be added to collections or nodes, which is an opportunity to add metadata that is only valid for a node in the context of a collection (e.g. rendering settings).
+Images may be added as nodes to multiple collections.
+
+
+### Metadata
+
+This RFC defines two main objects for OME-Zarr: `Collection`, `CollectionNode`. 
```

Comment:
A `CollectionNode` can be a `Collection`, so it's a bit confusing to say that these are two objects unless you explain that "object" here means something like "interface" or "protocol"

Reply:
What would be the best term here? Is it a class?

Reply:
actually I was wrong, regular inheritance isn't problematic for `Collection` and `Multiscales` -- there's a base `Node`, `Collection` and `Multiscales` (and anything else) inherit from `Node` (totally fine for them to add new attributes as children).

As for the requirement that there be only 1 root node, I don't think that can be expressed in a type system easily as long as the root is structurally identical to a `Collection`, but that can be added as a regular requirement

Reply:
If the requirement is just that the root node _have_ `version` (weaker than requiring that _only_ the root node have `version`), then this is a bit simpler.

Reply:
> Presumably `bioformats2raw.layout` and `plate` collections will still be around (not removed with this proposal). So a `Node` could be `Collection` or `Multiscales` or `bioformats2raw` or `plate`?

The idea is to remove `bioformats2raw.layout` and `plate` as separate entities with this proposal and express the functionality through attributes in the collection nodes. We need to work more on these.